lua/sg/cody/rpc.lua

```diff
@@ -170,6 +175,25 @@ M.request = async.wrap(function(method, params, callback)
   end)
 end, 3)
 
+---@return string | nil
+local function get_current_repo_url()
```

Comment:
I have something like this in lua/sg/cody/context.lua, so we should probably just consolidate them somewhere